xpra/client/mixins/window_manager.py

```diff
@@ -759,6 +759,10 @@ def _window_icon_image(self, wid, width, height, coding, data):
             if first_time("window-icons-require-pillow"):
                 log.info("showing window icons requires python-pillow")
             return None
+        if hasattr(Image, 'Resampling'):
+            from PIL.Image.Resampling import LANCZOS
+        else:
+            from PIL.Image import LANCZOS
```

Comment:
Can you please amend to use the same form as this one:
https://github.com/Xpra-org/xpra/blob/2cbcaf86fa1dfaf8fc7666e154af99d57646301f/xpra/codecs/pillow/encoder.py#L23-L35
Rationale: [The “Easier to Ask Forgiveness Than Permission” (EAFP) Style](https://realpython.com/python-lbyl-vs-eafp/#the-easier-to-ask-forgiveness-than-permission-eafp-style)

Reply:
> Rationale: [The “Easier to Ask Forgiveness Than Permission” (EAFP) Style](https://realpython.com/python-lbyl-vs-eafp/#the-easier-to-ask-forgiveness-than-permission-eafp-style)

I expected this :smile: But I then I didn't change to try/except because I wanted to wait first if you care at all about Ubuntu 20.04.